docs/third-party-developers/extensibility/data-store/checkout.md

```diff
@@ -0,0 +1,108 @@
+# wc/store/checkout
+
+## Table of Contents
+
+-   [Selectors](#selectors)
+    -   [getCustomerId](#getcustomerid)
+    -   [getCheckoutState](#getcheckoutstate)
+    -   [getOrderNotes](#getordernotes)
+    -   [hasError](#haserror)
+    -   [hasOrder](#hasorder)
+    -   [isAfterProcessing](#isafterprocessing)
+    -   [isBeforeProcessing](#isbeforeprocessing)
+    -   [isCalculating](#iscalculating)
+    -   [isComplete](#iscomplete)
+    -   [isIdle](#isidle)
+    -   [isProcessing](#isprocessing)
+
+## Selectors
+
+### getCustomerId
+
+Returns the ID of the customer.
+
+#### _Returns_
+
+-   `number`: ID of the customer.
+
+### getOrderNotes
+
+Returns the order notes.
+
+#### _Returns_
+
+-   `string`: Order notes.
+
+### hasError
+
+Returns true if an error occurred, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an error occurred.
+
+### hasOrder
+
+Returns true if an order exists, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order occurred.
```

Comment:
Could we change this to be something like "Whether the checkout is processing an existing order" cc @alexflorisca for wording ideas here

Reply:
It's tricky because in most of the descriptions here you are saying what the return value is. So I think we should change the descriptions to be more generic (for example here, it could be "Has a draft order been created?" and the return value description to something like "true if order exists, false otherwise". You can look [here](https://github.com/woocommerce/woocommerce-blocks/blob/feature/data-store-refactor/assets/js/data/checkout/default-state.ts#L12-L34) for inspiration with the descriptions

Reply:
Thanks for your reply, @alexflorisca. In https://github.com/woocommerce/woocommerce-blocks/issues/6945, @opr
mentioned https://developer.wordpress.org/block-editor/reference-guides/data/data-core-editor/ as an example. For consistency-reasons, I followed their style. This is the way they document `has***` functions:

> **hasChangedContent**
>
> Returns true if content includes unsaved changes, or false otherwise.
> 
> _Parameters_
> 
> - state **Object**: Editor state.
> 
> _Returns_
> 
> - **boolean**: Whether content includes unsaved changes.

With that in mind, I changed the description from `Returns true if an order exists, and false otherwise.` to `Returns true if a draft order had been created, and false otherwise.` and the return value description from `boolean: Whether an order occurred.` to `boolean: Whether a draft order had been created.`.

Would you mind doing another review?